src/gui/DatabaseOpenWidget.cpp

```diff
@@ -372,6 +369,7 @@ void DatabaseOpenWidget::pollYubikey()
     m_ui->checkChallengeResponse->setChecked(false);
     m_ui->comboChallengeResponse->setEnabled(false);
     m_ui->comboChallengeResponse->clear();
+    m_ui->comboChallengeResponse->addItem(tr("-- SELECT --"), -1);
```

Comment:
Do we really need the checkboxes after all this?

Reply:
Technically we do because people could have an "empty password" as their password. This is possible in KeePass2.

Reply:
Forgot about that. I actually consider that a conceptual bug in KeePass. But so be it.